packages/dataviews/src/field-types/datetime.tsx

```diff
@@ -36,7 +36,7 @@ export default {
 		return field.hasElements ? (
 			<RenderFromElements item={ item } field={ field } />
 		) : (
-			field.getValue( { item } )
+			new Date( field.getValue( { item } ) ).toLocaleString()
```

Comment:
If the date is undefined we will render "Invalid date" not sure if we want to render an empty string instead?
<img width="1208" height="449" alt="Screenshot 2025-10-30 at 09 34 15" src="https://github.com/user-attachments/assets/107d5d25-872d-44a6-8f1e-c2e7b72b5569" />

When the date is empty it is not that it is invalid, it may even be expected, but I guess we can also leave to the consumers the decision on how to handle this case, as they are the ones who know the meaning of an empty date.